src/agent/src/cdh.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+// Confidential Data Hub client wrapper.
+// Confidential Data Hub is a service running inside guest to provide resource related APIs.
+// https://github.com/confidential-containers/guest-components/tree/main/confidential-data-hub
+
+use anyhow::Result;
+use derivative::Derivative;
+use protocols::{
+    sealed_secret, sealed_secret_ttrpc_async, sealed_secret_ttrpc_async::SealedSecretServiceClient,
+};
+const CDH_ADDR: &str = "unix:///run/confidential-containers/cdh.sock";
+const CDH_TIMEOUT: i64 = 50 * 1000 * 1000 * 1000;
```

Comment:
IIUC from the (a little poor) documentation https://docs.rs/ttrpc/latest/ttrpc/context/struct.Context.html, this should be in nanoseconds. So leave a comment please.

Also rename it to something like `CDH_UNSEAL_SECRET_TIMEOUT`, more meaningful.